alvr/server/src/connection.rs

```diff
@@ -643,9 +644,24 @@ fn try_connect(mut client_ips: HashMap<IpAddr, String>) -> ConResult {
     };
 
     let tracking_manager = Arc::new(Mutex::new(TrackingManager::new()));
+    let hand_gesture_manager = Arc::new(Mutex::new(HandGestureManager::new()));
 
     let tracking_receive_thread = thread::spawn({
         let tracking_manager = Arc::clone(&tracking_manager);
+        let hand_gesture_manager = Arc::clone(&hand_gesture_manager);
+
+        let mut gestures_button_mapping_manager = Switch::as_option(
+            &SERVER_DATA_MANAGER.read().settings().headset.controllers,
+        )
+        .and_then(|config| {
+            Switch::as_option(&config.hand_tracking.use_gestures).and_then(|_| {
```

Comment:
Sorry, I was not completely clear, you should prefer using the method call syntax: `<something>.as_option()`